src/components/Images.vue

```diff
@@ -21,34 +21,74 @@
  -->
 
 <template>
-	<ImageEditor v-if="editing"
-		:mime="mime"
-		:src="src"
-		:fileid="fileid"
-		@close="onClose" />
-
-	<img v-else-if="data !== null"
-		:alt="alt"
-		:class="{
-			dragging,
-			loaded,
-			zoomed: zoomRatio !== 1
-		}"
-		:src="data"
-		:style="imgStyle"
-		@error.capture.prevent.stop.once="onFail"
-		@load="updateImgSize"
-		@wheel="updateZoom"
-		@dblclick.prevent="onDblclick"
-		@mousedown.prevent="dragStart">
+	<div class="image_container">
+		<ImageEditor v-if="editing"
+			:mime="mime"
+			:src="src"
+			:fileid="fileid"
+			@close="onClose" />
+
+		<img v-else-if="data !== null && !livePhoto"
+			ref="image"
+			:alt="alt"
+			:class="{
+				dragging,
+				loaded,
+				zoomed: zoomRatio !== 1
+			}"
+			:src="data"
+			:style="imgStyle"
+			@error.capture.prevent.stop.once="onFail"
+			@load="updateImgSize"
+			@wheel="updateZoom"
+			@dblclick.prevent="onDblclick"
+			@mousedown.prevent="dragStart">
+
+		<video v-else-if="data !== null && livePhoto"
+			ref="video"
+			:class="{
+				dragging,
+				loaded,
+				zoomed: zoomRatio !== 1
+			}"
+			:style="imgStyle"
+			:playsinline="true"
+			:poster="data"
+			:src="livePhotoSrc"
+			preload="metadata"
+			@canplaythrough="doneLoadingLivePhoto"
+			@loadedmetadata="updateImgSize"
+			@wheel="updateZoom"
+			@error.capture.prevent.stop.once="onFail"
+			@dblclick.prevent="onDblclick"
+			@mousedown.prevent="dragStart"
+			@ended="stopLivePhoto" />
+		<div v-if="livePhoto"
+			class="live-photo_play_button"
+			:class="{'live-photo_play_button--disabled': livePhotoCanBePlayed}"
+			:style="{left: `calc(50% - ${width/2}px)`}"
+			@pointerenter="playLivePhoto"
+			@pointerleave="stopLivePhoto">
+			<PlayCircleOutline v-if="livePhotoCanBePlayed" />
+			<NcLoadingIcon v-else />
+			{{ t('viewer', 'LIVE') }}
+		</div>
```

Comment:
This would be an a11y issue. As an interactive element, this should be a native button that can be accessed by keyboard, receive focus, have an actual `disabled` state. 

And probably a description. Not sure if "Live" describes well what this button does and that it works also on hover.

Reply:
I change it to `<button>`
Added the `focus` and `blur` event listeners
Added an `aria-description`, is it enough?
Added the disabled state. Are you sure that it makes sense? I can not be clicked, but it is still interactive.

I could add an event listener for the `click` event.

Reply:
> Added the disabled state. Are you sure that it makes sense? I can not be clicked, but it is still interactive.

Maybe I didn't understand what `livePhotoCanBePlayed` means. Does `live-photo_play_button--disabled` class display the button as disabled? And the button doesn't work when `livePhotoCanBePlayed` is falsy?

Reply:
Ah sorry, forgot to remove that class when I switch to simply having a loading icon.
I'll set the disabled state when `livePhotoCanBePlayed` is false

Reply:
Thanks!